test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -96,12 +94,11 @@ public void getControllerInfo() {
 
     @After
     public void tearDown() {
```

Comment:
The `@Before` method of the test starts a controller instance with name `multicontroller` . In case of a global timeout due to `@Rule`,  the `@After` method might not be invoked leading to an extra controller running on the test cluster. This will cause the subsequent tests to fail due to a resource issue in scaling/deploying controllers on the cluster.

Reply:
We keep the method timer in this test as an exception.